src/registrar/management/commands/load.py

```diff
@@ -13,11 +13,8 @@ class Command(BaseCommand):
     def handle(self, *args, **options):
         # django-auditlog has some bugs with fixtures
         # https://github.com/jazzband/django-auditlog/issues/17
-        if settings.DEBUG:
```

Comment:
I do actually recall this getting added with the fixtures change... This makes sense. This raises the question of how we should handle fixture data for production environments, since we probably don't want to be loading this in prod. Regardless, this is outside this PR imo